airflow-core/src/airflow/api_fastapi/logging/decorators.py

```diff
@@ -81,6 +81,7 @@ async def log_action(
     ):
         """Log user actions."""
         event_name = event or request.scope["endpoint"].__name__
+        skip_dry_run_events = {"clear_dag_run", "post_clear_task_instances"}
```

Comment:
We might want to add `patch_task_instance_dry_run_by_map_index`, `patch_task_instance_dry_run`, `post_clear_task_instances` ``create_backfill_dry_run`